ui/hooks/useTokensToSearch.js

```diff
@@ -96,7 +95,7 @@ export function useTokensToSearch({
     conversionRate,
     currentCurrency,
     chainId,
-    shuffledTokensList,
+    tokenList,
```

Comment:
Should `shuffledTokensList` be removed as an arg to `useTokensToSearch` and replaced with `tokenList` on line 108?

Reply:
🤔 weird. Alright, won't block on this.